package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vocabulary-tag",
-  "version": "2.0.0",
+  "version": "1.1.0",
```

Comment:
downgrading versions? evil

Reply:
Updated to version 2.1.0, the correct one! Thanks for pointing out :)